### PR TITLE
Update tutorial coding

### PR DESCRIPTION
**Pull request by mervey45**

### CODE REVIEW

**Comment by jmmargo** [comment]
Please resolve the conflicts.

**Comment by jmmargo** [comment]
Please resolve the conflicts

File: tutorials/abap-environment-behavior-action/abap-environment-behavior-action.md [hunk 1]
```diff
@@ -760,15 +747,13 @@ In this tutorial, wherever XXX appears, use a number (e.g. 000).
       ![Enhance behavior definition for projection view](projection5.png)
 
       **Hint: If your UI buttons don't appear, please add a space anywhere you want in your data definition `ZI_TRAVEL_M_XXX` and behavior definition `ZI_TRAVEL_M_XXX`, activate both and open your SAP Fiori preview again. It takes 120 seconds to make the buttons visible on the user interface.**
```
Comment (MichaelCzcz) [inline tutorials/abap-environment-behavior-action/abap-environment-behavior-action.md:749]: This should be formatted as a note.

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by mervey45** [comment]
Hi,

I resolved the conflict, do I need to commit the merge as well?

Best regards, 
Merve

**Comment by jmmargo** [comment]
Yes, please.

**Comment by mervey45** [comment]
Hi Joshua, 

I committed the changes.

Best regards, 
Merve

File: tutorials/abap-environment-behavior-action/abap-environment-behavior-action.md (hunk 1 above)
Comment (jmmargo) [reply tutorials/abap-environment-behavior-action/abap-environment-behavior-action.md]: Merve, did you already incorporate this suggestion regarding the note?  I did not see it in the changed files and wanted to be sure before I merged it.

Comment (mervey45) [reply tutorials/abap-environment-behavior-action/abap-environment-behavior-action.md]: Yes, I formatted the hint section.

**Pull request merged**